indico/MaKaC/webinterface/tpls/latex/report.tpl

```diff
@@ -30,8 +30,8 @@
     % for item in items:
         \newpage
         \fancyhead[L]{\small \rmfamily \color{gray} \truncateellipses{${conf.getTitle() | latex_escape}}{80pt} / ${title}}
-        \fancyhead[R]{\small \rmfamily \color{gray} \truncateellipses{${item.getTitle() | latex_escape}}{150pt}}
-        \addcontentsline{toc}{section}{${item.getTitle() | latex_escape}}
+        \fancyhead[R]{\small \rmfamily \color{gray} \truncateellipses{${item.title or item.getTitle() | latex_escape}}{150pt}}
+        \addcontentsline{toc}{section}{${item.title or item.getTitle() | latex_escape}}
```

Comment:
why do we need this? also, I think the `|latex_escape` is only applied to `item.getTitle()`...